wavefront_pyformance/wavefront_reporter.py

```diff
@@ -41,13 +44,32 @@ def report_now(self, registry=None, timestamp=None):
         registry = registry or self.registry
         metrics = registry.dump_metrics()
         for key in metrics.keys():
+            metric_name, metric_tags = self.decode_key(key)
+            tags = self.tags
+            if metric_tags:
+                tags = self.tags.copy()
+                tags.update(metric_tags)
+
+            wf_hist = None
+            if key in registry._histograms:
```

Comment:
We should avoid directly using private variables within the registry (these could change, may not be safe to do concurrent access etc).

You can add a `isHistogram(...)` function that essentially looks for certain keys to exist: https://github.com/omergertel/pyformance/blob/b71056eaf9af6cafd3e3c4a416412ae425bdc82e/pyformance/registry.py#L145 in the values dict and establishes whether this is a histogram and not a timer.

Once that is verified, you can just invoke `registry.histogram(key)` to get the instance.